ast/visit.go

```diff
@@ -559,8 +561,9 @@ func (vis *VarVisitor) visit(v interface{}) bool {
 	}
 	if vis.params.SkipRefHead {
 		if r, ok := v.(Ref); ok {
-			for _, t := range r[1:] {
-				vis.Walk(t)
+			rSlice := r[1:]
+			for i := range rSlice {
```

Comment:
💭 Wouldn't this work the same?
```go
for i := range r[1:] {
    vis.Walk(r[i+1])
}
```

but it's all the same. The `rSlice := r[1:]` doesn't cost us much, I presume.